src/opa.ts

```diff
@@ -28,6 +28,11 @@ export function canUseBundleFlags(): boolean {
     return installedOPASameOrNewerThan("0.14.0-dev") && bundleMode;
 }
 
+export function canUseStrictFlag(): boolean {
+    let strictMode = vscode.workspace.getConfiguration('opa').get<boolean>('strictMode', true);
+    return strictMode && installedOPASameOrNewerThan("0.37.0-dev");
```

Comment:
Still accurate? I'm thinking
```suggestion
    return strictMode && installedOPASameOrNewerThan("0.37.0");
```